src/Map.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Check comment above.

Reply:
Yes, git your point and removed the unnecessary imports